debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgproto/PgProtoReplicationMessage.java

```diff
@@ -70,8 +70,13 @@ public Operation getOperation() {
 
     @Override
     public Instant getCommitTime() {
-        // value is microseconds
-        return Instant.ofEpochSecond(0, rawMessage.getCommitTime() * 1_000);
+        Long commitTime = rawMessage.getCommitTime();
+        if(commitTime != null) {
+            // value is microseconds
+            return Instant.ofEpochSecond(0, commitTime * 1_000);
+        } else {
```

Comment:
`rawMessage.getCommitTime();` above returns a primitive `long`,  i.e. the `Long commitTime` actually will never be `null` and this branch is dead code. It seems the original version was just fine?

Reply:
You know this code better than me, Ill revert it